### ISSUE TITLE
Adding Tech Stack to README

### ISSUE THREAD

**Issue by shravya312** [Devanik21/ISRO_Mining_Site_FINAL_APP#9]

**Comment by shravya312** [comment]
Hi @Devanik21 Thank you for assigning this task to me. Kindly can you add the labels gssoc-extd, hacktoberfest-accepted, hacktoberfest and level-1.

**Comment by Devanik21** [comment]
Thanks ! and please update this for more information.

Possible solution-

Tech Stack
Languages: Python, HTML, CSS
Frameworks: Streamlit for web app development
Libraries: Pandas, NumPy, Scikit-learn, Matplotlib, Seaborn, Plotly, XGBoost, LightGBM
Tools: Joblib for model serialization, Tesseract for OCR (optional), TensorFlow & Keras for AI/ML models
Version Control: GitHub for code management

**Comment by AnujSaha0111** [comment]
I want to work on it. Can you please assign it to me?

**Comment by Absk024** [comment]
i would like to contribute @Devanik21

**Comment by Absk024** [comment]
i would like to contribute @Devanik21
